src/NovelRT/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ find_package(spdlog ${NOVELRT_SPDLOG_VERSION} REQUIRED)
 find_package(TBB ${NOVELRT_ONETBB_VERSION} REQUIRED)
 find_package(Vorbis ${NOVELRT_VORBIS_VERSION} REQUIRED)
 find_package(Vulkan ${NOVELRT_VULKAN_VERSION} REQUIRED)
```

Comment:
GLFW pulls in Vulkan based on how we have it set up, so this appears to link properly.

Reply:
.... on Macs. Not on Windows, so this will properly refer to the Vulkan Loader conan package on those.